vars/loadResources.groovy

```diff
@@ -0,0 +1,16 @@
+def call(Map args = [:]) {
+    if(!args.file) {
+        error "Missing manadatory parameter: file"
+    }
+
+    def resources = readYaml file: args.file
+
+    def resourcesList = [:]
+    if(resources.getClass().getSimpleName() == "LinkedHashMap") {
```

Comment:
can we use `instanceof` ?
Also `get`**`Foobar`**`()` can be accessed as property `foobar` in groovy. So this would become 
`resources.class.simpleName`